### PR TITLE
feat: add webpack.config.dev.js for local serving of remoteEntry.js

### PR DESCRIPTION
**Pull request by khanti42**
Local serving of remoteEntry.js can be done with 

`yarn serve:fed` from the `packages/get-starknet` sub folder. 
Associated with https://github.com/Consensys/get-starknet/pull/13

### CODE REVIEW

File: packages/get-starknet/webpack.config.dev.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { merge } = require('webpack-merge');
+const common = require('./webpack.common.js');
```
Comment (stanleyyconsensys) [inline packages/get-starknet/webpack.config.dev.js:3]: add // eslint-disable-next-line @typescript-eslint/no-var-requires for each require

**Review by stanleyyconsensys — CHANGES_REQUESTED** [review]
if we wanna setup a complete local dev env
we also need to make this get-starknet wrapper using a SNAP from local rather from prod

in order to do that
update `MetaMaskSnapWallet` class to change the const SNAPI_ID reading from env var , default is `npm:@consensys/starknet-snap`

file: wallet.ts
```javascript
private static readonly SNAPI_ID =  process.env.SNAP_ID ?? npm:@consensys/starknet-snap
```

file: .env
```bash
SNAPI_ID=local:http://localhost:8081
```

**Review by stanleyyconsensys — APPROVED** [review]
LGTM

**Comment by khanti42** [comment]
Check  [PR-249](https://github.com/Consensys/starknet-snap/pull/249) for update.

**Pull request closed without merging**